.github/workflows/test.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: pytest --capture=no --cov --cov-report=xml .
 
       - name: codacy-coverage-reporter
-        if: ${{ matrix.os == 'ubuntu-latest' }}
+        if: ${{ matrix.os == 'ubuntu-latest' && github.event_name == 'push' }}
```

Comment:
Don't we also want to trigger on PRs to see how they affect coverage?

Reply:
Yes!

I was trying to figure this out, but for some reason on pr triggers the repo secrets are not accessible. I'll look into a way around that

Reply:
I think you have to use the [less secure `pull_request_target`](https://securitylab.github.com/research/github-actions-preventing-pwn-requests) which is fine here I think cause who cares if someone gets the Codacy token. 😆

Reply:
yeah, was just reading that explanation!